src/www/ui/template/ui-clearing-view_bulk.html.twig

```diff
@@ -135,6 +135,15 @@
                     </label>
                   </div>
                 </div>
+                <div class="col-sm-auto pr-0">
+                  <div class="form-check form-check-inline">
+                    <input type="checkbox" class="form-check-input" id="scanOnlyFindings"/>
```

Comment:
Missing the `name` attribute causes this flag to be not passed with the form request.
```suggestion
                    <input type="checkbox" class="form-check-input" id="scanOnlyFindings" name="scanOnlyFindings" />
```